cni/pkg/nodeagent/informers.go

```diff
@@ -207,13 +207,10 @@ func (s *InformerHandlers) reconcilePod(input any) error {
 		}
 		wasAnnotated := oldPod.Annotations != nil && oldPod.Annotations[constants.AmbientRedirection] == constants.AmbientRedirectionEnabled
 		isAnnotated := newPod.Annotations != nil && newPod.Annotations[constants.AmbientRedirection] == constants.AmbientRedirectionEnabled
-		isOpOut := newPod.Annotations != nil && newPod.Annotations[constants.AmbientRedirection] == constants.AmbientRedirectionDisabled
```

Comment:
do we need similar check for label?

Reply:
No, because it's already take care of by the other checks - before we had to check label + annotation opt-out.

Now that the label is either `none` or `ambient`, you can't have both opt out and enable at the same time for a pod, so the logic here is simpler.

This is also well-covered in unit tests.